components/echo/echo.toit

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2024 Toitware ApS.
+// Use of this source code is governed by a Zero-Clause BSD license that can
+// be found in the tests/LICENSE file.
+
+/**
+Example code for communicating with the external echo service.
+*/
+
+import system.external
+
+main:
+  echo := external.Client.open "toitlang.org/demo-echo"
+  echo.set-on-notify:: print "Got message: $it.to-string"
+  echo.notify "Hello, world!"
+
+  function-id := 0
```

Comment:
Maybe pull this out as a constant? That feels simpler than this way of naming the argument.

Reply:
done.